inference-engine/src/inference_engine/ie_plugin_cpp.hpp

```diff
@@ -92,6 +92,10 @@ class InferencePlugin {
         PLUGIN_CALL_STATEMENT(return actual->LoadNetwork(modelPath, config));
     }
 
+    ExecutableNetwork ToExecutableNetwork(const std::shared_ptr<IExecutableNetworkInternal>& impl) {
+        return ExecutableNetwork(impl, actual);
+    }
```

Comment:
could you please update based on https://github.com/openvinotoolkit/openvino/pull/5645 ?

Reply:
Done, btw, CI is broken currently (probably by #5624, going to be fixed by #5697)